test/langtools/jdk/jshell/CompletionSuggestionTest.java

```diff
@@ -810,4 +810,15 @@ public void testDuplicateImport() {
                                      ste(m2, RECOVERABLE_DEFINED, VALID, true, MAIN_SNIPPET));
         assertSignature("test(|", "void test(String s)", "void test(Integer i)");
     }
+
+    //JDK-8326333
```

Comment:
Such simple bug-id seems unnecessary or uncomplete.  It is better to privode some description content.